clients/src/main/resources/common/message/MetadataRequest.json

```diff
@@ -31,7 +31,10 @@
     // Starting in version 8, authorized operations can be requested for cluster and topic resource.
     //
     // Version 9 is the first flexible version.
-    // Version 10 add topicId
+    //
+    // Version 10 adds topicId.
+    //
+    // Version 11 deprecates IncludeClusterAuthorizedOperations field (KIP-700).
```

Comment:
How about saying this function is migrated to ```DescribeClusterRequest```?

Reply:
Yeah, why not. I will add it.